src/jit/bitsetasshortlong.h

```diff
@@ -664,9 +664,11 @@ void BitSetOps</*BitSetType*/ BitSetShortLongRep,
                /*BitSetTraits*/ BitSetTraits>::ClearDLong(Env env, BitSetShortLongRep& bs)
 {
     assert(!IsShort(env));
-    // Recall that ClearD does *not* require "bs" to be of the current epoch.
-    // Therefore, we must allocate a new representation.
-    bs = MakeEmptyArrayBits(env);
```

Comment:
> But It appeared that all current users of ClearD do not change epoch.

I see twenty uses of `ClearD` in the compiler today:
```
src/jit/assertionprop.cpp:4559:    BitVecOps::ClearD(apTraits, fgFirstBB->bbAssertionIn);
src/jit/codegenlegacy.cpp:12585:        VarSetOps::ClearD(compiler, compiler->compCurLife);
src/jit/copyprop.cpp:299:        VarSetOps::ClearD(this, optCopyPropKillSet);
src/jit/emit.cpp:1475:    VarSetOps::ClearD(emitComp, emitInitGCrefVars);
src/jit/emit.cpp:1476:    VarSetOps::ClearD(emitComp, emitPrevGCrefVars);
src/jit/emit.cpp:4567:    VarSetOps::ClearD(emitComp, emitThisGCrefVars); // This is initialized to Empty at the start of codegen.
src/jit/flowgraph.cpp:1818:        // Initialize the per-block bbReach sets. (Note that we can't just call BlockSetOps::ClearD()
src/jit/flowgraph.cpp:10014:        BlockSetOps::ClearD(this, bNext->bbReach);
src/jit/liveness.cpp:489:        VarSetOps::ClearD(this, fgCurUseSet);
src/jit/liveness.cpp:490:        VarSetOps::ClearD(this, fgCurDefSet);
src/jit/liveness.cpp:725:            VarSetOps::ClearD(this, inScope);
src/jit/liveness.cpp:915:        VarSetOps::ClearD(this, initVars);
src/jit/liveness.cpp:1151:        VarSetOps::ClearD(m_compiler, m_liveOut);
src/jit/liveness.cpp:1265:            VarSetOps::ClearD(m_compiler, m_liveIn);
src/jit/liveness.cpp:1266:            VarSetOps::ClearD(m_compiler, m_liveOut);
src/jit/lsra.cpp:1741:    BlockSetOps::ClearD(compiler, bbVisitedSet);
src/jit/lsra.cpp:9317:            VarSetOps::ClearD(compiler, sameResolutionSet);
src/jit/lsra.h:1109:        BlockSetOps::ClearD(compiler, bbVisitedSet);
src/jit/regalloc.cpp:1343:            VarSetOps::ClearD(this, varAsSet);
src/jit/regalloc.cpp:6351:            VarSetOps::ClearD(this, raLclRegIntf[i]);
```

How did you verify that the epoch of each bitset does not change between the creation of each bitset and the point at which they are cleared? If the epoch has changed, the failure mode could be pretty bad, as if the number of bits has increased we could end up scribbling over arbitrary memory.